frigg_worker/jobs.py

```diff
@@ -84,7 +84,7 @@ def succeeded(self):
     def settings(self):
         return build_settings(self.working_directory, self.docker)
 
-    def clone_repo(self, depth=1):
+    def clone_repo(self, depth=20):
```

Comment:
the question might be, what do we do if the commit is not found? Clone again with a higher number?